### ISSUE TITLE
hadolint fails when use apk add with --repository

### ISSUE THREAD

**Issue by BrokeRU** [hadolint/hadolint#290]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Run `apk add` with `--repository` argument.

### Actual behavior
Hadolint expects equal sign after `--repository` argument even if `apk` doesn't require it.
I have to place '=' after `--repository` to avoid `DL3018` error

### Steps to reproduce the behavior
```
FROM node:10.14.2-alpine
RUN apk add --no-cache --repository https://nl.alpinelinux.org/alpine/edge/testing flow=0.78.0-r0
```

```
DL3018 Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.15.0-6-gaeb6041
```
